### DISCUSSION TITLE
Welcome to russia-must-be-stopped Discussions!

### DISCUSSION THREAD

**Discussion by nothing-much-coder** [codergator/DDoS-Russia-in-4-easy-steps#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Puintin** [comment]
Hi, Thanks for making this possible! I am from the Netherlands and have it running on 3 laptops that I had lying around.
I have also shared your work with friends and twitter-threads.
Questions: will you be updating the list of websites and or can I update them myself?
Do you have any idea how many instances are running?
Is there a way to tell it is effective other then visiting the targeted sites?

Cheers

**Reply by nothing-much-coder** [reply]
Awesome — I'm really glad to hear that the repo helped you get up and running. I'm totally open to PR's, and if you look at the commits, you'll see I've been updating it. If you join the IT Army of Ukraine Telegram group, you'll see that they periodically post the results of the DDoS'ing that we're all doing, from all sources.

<img width="468" alt="Screen Shot 2022-03-08 at 3 45 33 PM" src="https://user-images.githubusercontent.com/16888908/157338515-3fba2d4b-0965-4d63-b057-eea5fb3f0402.png">
 
That's the group that I use to keep up with the latest targets to focus on.

https://t.me/itarmyofukraine2022

And nope, I don't know how many others are actually doing this; it's all very decentralized.

**Comment by Puintin** [comment]
Hi,

Thanks for the reply.
I have joined the telegram group.
I will keep my spare laptops running the script for as long as
needed/useful.

Cheers

Op di 8 mrt. 2022 om 23:47 schreef The Codergator ***@***.***
